mrmustard/physics/utils.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains helper functions for physics.
+"""
+import numpy as np
+
+from mrmustard import math
+
+
+def real_gaussian_integral(
+    Abc: tuple,
+    idx: tuple[int, ...],
+):
+    r"""Computes the Gaussian integral of the exponential of a real quadratic form.
+    The integral is defined as (note that in general we integrate over a subset of m dimensions):
+
+    :math:`\int_{R^m} F(x) dx`
```

Comment:
```suggestion
    :math:`\int_{R^m} F(x) dx\:,`
```
If you want this to be in the middle of a page like an equation on a paper, you'll need to replace `math` with `.. math::` (I believe)